packages/util/src/func.js

```diff
@@ -0,0 +1,15 @@
+export const debounce = (func, delay) => {
+  if (typeof func !== 'function') {
```

Comment:
Since this util package is one used internally by the DS, I'd prefer to leave these in-src usage helps out.  My main reason would be to rely on us as expert users of the DS and not ship this code to consumers of things like the Tooltip component.

Reply:
Makes sense - I can adjust